lib/renderer/ipc-renderer-internal.ts

```diff
@@ -20,3 +20,9 @@ ipcRendererInternal.sendTo = function (webContentsId, channel, ...args) {
 ipcRendererInternal.sendToAll = function (webContentsId, channel, ...args) {
   return ipc.sendTo(internal, true, webContentsId, channel, args)
 }
+
+ipcRendererInternal.invoke = async function<T> (channel: string, ...args: any[]) {
+  const { error, result } = await ipc.invoke<T>(internal, channel, args)
+  if (error) { throw new Error(`Error invoking remote method '${channel}': ${error}`) }
```

Comment:
The throw here should be on a new line

Reply:
I can do that, I kept the same formatting as https://github.com/electron/electron/blob/master/lib/renderer/api/ipc-renderer.ts#L26

Reply:
done